transformer_engine/jax/layernorm.py

```diff
@@ -49,15 +50,18 @@ def layernorm(inputs: jnp.ndarray,
     layernorm_type = canonicalize_layernorm_type(layernorm_type)
     if layernorm_type == 'rmsnorm':
         assert beta is None, "beta should be None if layernorm_type is 'rmsnorm'"
+        if zero_centered_gamma:
+            raise NotImplementedError
```

Comment:
`NotImplementedError` isn't the right exception because we never plan to implement this case:
```suggestion
        assert not zero_centered_gamma, "zero_centered_gamma is not supported if layernorm_type is 'rmsnorm'"
```